elastic/kibana
# [ResponseOps][Alerting] Connectors editors dark-mode dialogs fix-ups

Resolves https://github.com/elastic/kibana/issues/139234

## Summary
Browsers autofill password fields, and when that happens they add a styling to inputs that can't be overridden. This is why we were seeing a light mode style in dark mode. To remove this style I added `autocomplete="new-password"` to password fields and `autocomplete="off"` to the other inputs that the browser was trying to autofill in the connector flyout. 

### Checklist


### To verify

- Enable dark mode in the [advanced settings](http://localhost:5601/app/management/kibana/settings)
- Create a new connector that would have authentication. Verify that the username/password or id/api-key are not autopopulated and are in dark mode.
- Create an email connector and try to test it. Verify that the subject field does not suggest any inputs.
